package.json

```diff
@@ -30,35 +30,35 @@
     "audit": "better-npm-audit audit"
   },
   "peerDependencies": {
-    "@azure/msal-node": "*",
-    "cookie-session": "*",
+    "@azure/msal-node": "1 || 2",
```

Comment:
It'll work for both?

Reply:
But not for 3?

Reply:
Yes, it'll work for both, we've done a few upgrades from v1 to v2 without a problem. [See the release notes for v2](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/CHANGELOG.md#200).

3 doesn't exist yet. I find `*` a bit too permissive, I'd rather an `npm install` error to be a signal for us that we need to check whether it works with a new major version.